Week-1/solution2.css

```diff
@@ -0,0 +1,126 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Lato";
+}
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  overflow: hidden;
+  width: 100%;
+}
+.sidebar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 50px;
+  background-color: #f0f0f0;
+  z-index: 1010;
+}
+.sidebar > div {
+  padding-top: 20px;
+  height: 70%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+.sidebar > div > i {
+  font-size: 1rem;
+  font-weight: bold;
+}
+.section1 {
```

Comment:
If we have more than 5 sections, we will be writing section1, section2 and so on. Its a good practice to have a common name which can be reused.

Reply:
Since both section has different styling and no styles in common, I am not able to use common class but I have renamed it to more descriptive class name.